packages/helpers/qrcode-modal/src/browser/assets/style.css

```diff
@@ -318,10 +318,19 @@
   font-size: 15px;
 }
 
+.walletconnect-connect__buttons__wrapper {
+  max-height: 44vh;
```

Comment:
Generally the content of the buttons should determine the max height, if the button change height based n the screen it can cause responsive issues or overlapping.